core/lib/spree/testing_support/dummy_ability.rb

```diff
@@ -0,0 +1,10 @@
+class DummyAbility
+  include CanCan::Ability
+
+  attr_reader :user
+
+  def initialize(user = nil)
```

Comment:
There is no call side for the `DummyAbility` that passes in a user. Hence id argue to drop the argument altogether. And thus you could drop the full `#initialize` method.